src/main/src/main/java/org/geoserver/catalog/LayerInfo.java

```diff
@@ -168,4 +168,28 @@
      */
     void setAdvertised(boolean advertised);
 
+    /**
+     * The default interpolation method.
+     *
+     * <p>
+     * If not specifed (i.e. {@code null}), the service default will be used.
+     * </p>
+     */
+    String getDefaultInterpolationMethod();
```

Comment:
That's why I opted for a String property, I didn't want to either 1) create two identical enums or 2) run the risk of breaking backwards compatibility by refactoring WMSInfo into using a standalone WMSInterpolation.

Should I move on with 1), 2), or leave things as they are?